### PR TITLE
Issue 781 user badges without meta

### PR DESCRIPTION
**Pull request by koustov**
> First thing, PLEASE READ THIS: [ReactPlay Code Review Checklist](https://github.com/reactplay/react-play/wiki/ReactPlay-Code-Review-Checklist)

# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce.

# Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

# Screenshots or example output

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -9,6 +9,11 @@
     <meta name="theme-color" content="#000000" />
     <meta property="og:type" content="website" />
     <meta name="twitter:site" content="@ReactPlayIO" />
+    
+    <meta name="description" content="__META_DESCRIPTION__"/>
```
Comment (atapas) [inline public/index.html:13]: Why do we need these as the branch changes are without meta?

File: src/common/badges-dashboard/BadgeMap.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+export const BadgeMap = {
+  "Growling Panther": "winner",
```
Comment (atapas) [inline src/common/badges-dashboard/BadgeMap.js:2]: How do we plan to accomodate future badges in this structure?

For the upcoming event badges for example?

File: src/common/badges-dashboard/ClaimedBadges.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
+import BadgeCard from "./BadgeCard";
+
+const ClaimedBadges = ({
+  badges
+}) => {
+  return (
+    <>
+      {
+        badges.length === 0 ?
+          <p>No badges claimed yet</p>
```
Comment (atapas) [inline src/common/badges-dashboard/ClaimedBadges.js:10]: We can style this one better.

File: src/common/components/user-profile/user-profile.css [hunk 4]
```diff
@@ -0,0 +1,89 @@
+.snip1336 {
```
Comment (atapas) [inline src/common/components/user-profile/user-profile.css:1]: Why does the class name like this?

File: webpack.config.js [hunk 5]
```diff
@@ -0,0 +1,5 @@
+const node_externals = require("webpack-node-externals");
+
```
Comment (atapas) [inline webpack.config.js:2]: What is this for?

File: public/index.html (hunk 1 above)
Comment (koustov) [reply public/index.html]: removed

File: src/common/badges-dashboard/ClaimedBadges.js (hunk 3 above)
Comment (koustov) [reply src/common/badges-dashboard/ClaimedBadges.js:10]: Updated

File: webpack.config.js (hunk 5 above)
Comment (koustov) [reply webpack.config.js]: removed

File: src/common/components/user-profile/user-profile.css (hunk 4 above)
Comment (koustov) [reply src/common/components/user-profile/user-profile.css]: removed

File: src/common/badges-dashboard/BadgeMap.js (hunk 2 above)
Comment (koustov) [reply src/common/badges-dashboard/BadgeMap.js]: removed

**Pull request merged**